sql-cli/pyproject.toml

```diff
@@ -39,7 +39,7 @@ Jinja2 = ">=2.10.1, <=3.1"
 python-frontmatter = "^1.0.0"
 PyYAML = ">=5.4.1"
 python-dotenv = "^0.21.0"
-astro-sdk-python = {extras = ["all"], version = "^1.2.0"}
+astro-sdk-python = {extras = ["all"], version = "^1.2.2"}
```

Comment:
It would be great to add a comment to why we only support the latest version of the Astro Python SDK